ImportBlocks/ImportBlocks.cs

```diff
@@ -94,6 +118,46 @@ private bool OnExport(string[] args)
             return true;
         }
 
+        private bool ExportAppLog(Block block, FileStream fs)
+        {
+            foreach (Transaction tx in block.Transactions)
+            {
+                if (tx.Type != TransactionType.InvocationTransaction)
+                    continue;
+                
+                JArray _params = new JArray();
+                JString txHash = new JString(tx.Hash.ToString());
+                Console.Write($"Export tx {txHash}\n");
+
+                _params.Add(txHash);
+                JObject getAppLogRequest = new JObject();
+                getAppLogRequest["id"] = -1;
+                getAppLogRequest["method"] = "getapplicationlog";
+                getAppLogRequest["params"] = _params;
+                JObject txState = System.RpcServer.ProcessRequest(null, getAppLogRequest);
+
+                Console.Write($"Result is {txState["result"]}\n");
+                if (!txState["result"])
+                    if ((int)txState["code"].AsNumber() == -100 || (int)txState["code"].AsNumber() == -32602 || (int)txState["code"].AsNumber() == -32600)
```

Comment:
@erikzhang, @shargon, it would be nice to change this line for any kind of error.